outbound-dialer/src/main/java/org/motechproject/nms/outbounddialer/service/impl/TargetFileServiceImpl.java

```diff
@@ -113,13 +117,15 @@ public TargetFileServiceImpl(@Qualifier("outboundDialerSettings") SettingsFacade
                                  MotechSchedulerService schedulerService, AlertService alertService,
                                  SubscriptionDataService subscriptionDataService,
                                  CallRetryDataService callRetryDataService,
-                                 SubscriberDataService subscriberDataService) {
+                                 SubscriberDataService subscriberDataService,
```

Comment:
Should this use the subscriberDataService, or the subscriberService?  We've never come up with clear rules for when to use which.